frontend/src/components/pages/your_clubs/ClubList.tsx

```diff
@@ -169,13 +192,15 @@ function BookInfo(props: BookInfoProps) {
   const classes = useStyles();
 
   return (
-    <>
-      <p className={classes.textElement} style={{marginRight: 20}}>
-        <b>Current Book: </b><br/>
-        {props.book.title}<br/>
-        by {props.book.author}
+    <div>
+      <div className={classes.boldTextElement}>
+        Current Book:
+      </div>
+      <p className={classes.textElement}>
+        <div> {props.book.title} </div>
```

Comment:
nit: If you want an actual space here, use `&nbsp;` (non-breaking space). By default HTML removes 'extra' spaces in the code.

Reply:
Thank you for the tip!